cmd/acadock-monitoring/main_test.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/Scalingo/acadock-monitoring/cmd/acadock-monitoring/isgraceful"
+
+	"github.com/stretchr/testify/require"
+)
+
+// getCmd returns a command to run the main.go file
+func getCmd(t *testing.T, tmpdir string) *exec.Cmd {
+	binaryPath := tmpdir + "/main"
+
+	// Clean the temporary directory
+	_ = os.Remove(binaryPath)
+
+	// Build main.go
+	err := exec.Command("go", "build", "-o", binaryPath, "main.go").Run()
+	require.NoError(t, err)
+
+	return exec.Command(binaryPath)
+}
+
+func TestService_Shutdown(t *testing.T) {
+	// Set some environment variables
+	upgradeTimeout := time.Millisecond * 200
+	shutdownTimeout := time.Millisecond * 100
+	t.Setenv("GRACEFUL_UPGRADE_TIMEOUT", upgradeTimeout.String())
+	t.Setenv("GRACEFUL_SHUTDOWN_TIMEOUT", shutdownTimeout.String())
+
+	for _, s := range []os.Signal{syscall.SIGINT, syscall.SIGTERM} {
+		t.Run("Signal "+s.String(), func(t *testing.T) {
```

Comment:
issue(blocking): the `t.Run(...)` message seems not to explain what behavior is expected.
Can you make it clearer please.

Reply:
I can make this clearer, sure. Just so you know, this message is similar to that in go-utils/graceful/service_test.go:36

Reply:
main_test.go removed